spec/views/_footer.html.erb_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe "/_footer.html.erb", :type => :view do
+  describe '#nuBrand' do
+    before do
+      render
+    end
+    
+    describe 'copyright year' do
+      let(:this_yr) { Time.now }
+      let(:next_yr) { Time.now + 1.year }
+      
+      it 'shows this year' do
+        expect(rendered).to have_text("© #{this_yr.year} Northwestern University")
+        expect(rendered).to_not have_text("© #{next_yr.year} Northwestern University")
+      end
+      
+      it 'shows a year later' do
+        Timecop.freeze(next_yr) do
+          render
+          expect(rendered).to have_text("© #{next_yr.year} Northwestern University")
+        end
+      end
+    end
+  end
+end
```

Comment:
Needs and EOF :) I like that you're using view specs instead of capybara, it's much lighter.

Reply:
you mean for timecop?

Reply:
I mean for the file (`_footer.html.erb_spec.rb`).  See this link:
https://github.com/galterlibrary/digital-repository/pull/581/files/16d589ea1956032b34fa9d9ab7cb155d31764027#diff-ba8e672e8cb04fe32f6e7ea2e88b214e
The little `no new line at the end of file` badge. Just being my anal self.

Reply:
ahhh i see, haha. i try to keep that in mind. i forget too often it seems ha.